src/main/resources/assets/proton/templates/stairs_blockstate_template.json

```diff
@@ -0,0 +1,44 @@
+{
+  "variants": {
+    "facing=east,half=bottom,shape=straight":  { "model": "model_normal" },
```

Comment:
you should somehow indicate that this gets replaced, change it to $model_normal or whatever